Userland/Utilities/headless-browser.cpp

```diff
@@ -151,7 +181,7 @@ class Application : public WebView::Application {
     RefPtr<Requests::RequestClient> m_request_client;
     RefPtr<ImageDecoderClient::Client> m_image_decoder_client;
 
-    OwnPtr<HeadlessWebContentView> m_web_view;
+    Vector<NonnullOwnPtr<HeadlessWebContentView>> m_web_views;
```

Comment:
We currently refuse to handle "child web views" in the headless browser (window.open, etc). But I guess this can be addressed later when we add that to headless browser's application thingy. Just feel like there's a reason other chromes store them differently than a vector of NNOP.

Reply:
Not 100% sure, but I did take a quick look at that, and I think headless-browser mostly won't care about how we store the web views here. The thing that tracks page-id-to-web-view association is WebContentClient, which is a level below us here. Here, we will mostly just have to hook up `on_new_web_view` and `initialize_client` I think.